docker/docker-bootstrap.sh

```diff
@@ -41,7 +41,7 @@ if [[ "${1}" == "worker" ]]; then
   celery worker --app=superset.tasks.celery_app:app -Ofair -l INFO
 elif [[ "${1}" == "beat" ]]; then
   echo "Starting Celery beat..."
-  celery beat --app=superset.tasks.celery_app:app --pidfile /tmp/celerybeat.pid -l INFO
+  celery beat --app=superset.tasks.celery_app:app --pidfile /tmp/celerybeat.pid -l INFO -s /app/superset_home/celerybeat-schedule
```

Comment:
Should we pick the path to `/app/superset_home` from `${SUPERSET_HOME}`?

Reply:
Good point, changed it